src/app/space/create/deployments/apps/deployment-card.component.spec.ts

```diff
@@ -184,17 +202,22 @@ describe('DeploymentCardComponent async tests', () => {
       expect(item).toBeFalsy();
     }));
 
-    it('should invoke service \'delete\' function on Delete item click', fakeAsync(() => {
+    it('should call the delete modal open method', fakeAsync(()  => {
       const item: DebugElement = getItemByLabel('Delete');
       expect(item).toBeTruthy();
-      expect(mockSvc.deleteApplication).not.toHaveBeenCalled();
+      expect(mockSvc.deleteDeployment).not.toHaveBeenCalled();
       item.query(By.css('a')).triggerEventHandler('click', new CustomEvent('click'));
-
       fixture.detectChanges();
 
-      expect(mockSvc.deleteApplication).toHaveBeenCalled();
-      expect(notifications.message).toHaveBeenCalled();
+      expect(component.openModal).toHaveBeenCalled();
     }));
+
+    it('should call the delete service method when the modal event fires', fakeAsync(() => {
+      const de: DebugElement = fixture.debugElement.query(By.directive(FakeDeleteDeploymentModal));
+      de.componentInstance.deleteEvent.emit();
+      expect(component.delete).toHaveBeenCalled();
```

Comment:
Add a `expect(component.delete).not.toHaveBeenCalled()` at the top of this test as well so that it's clear that the event emission triggered the call. Rather than testing component.delete, would it make sense to test `mockSvc.deleteDeployment` being called instead? This promotes a little bit looser coupling IMO.